src/components/shared/typography/Para1.jsx

```diff
@@ -8,7 +8,7 @@ export default styled.p`
     Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
   font-style: normal;
   font-weight: normal;
-  font-size: 1.25rem;
-  line-height: 1.6;
+  font-size: 1.rem;
```

Comment:
Hey don't change the font sizes, there might be inconsistencies in the design, so let's stick to the original ones itself that we created. Also keep the line height to a constant 1.6 (no unit here).

Reply:
ok